framework/resources/Functional/no_result_set/DRILL_6834_use_no_result_set.sql

```diff
@@ -0,0 +1,3 @@
+set `exec.return_result_set_for_ddl` = false;
+use dfs.`drillTestDir`;
```

Comment:
Same here. Not sure about what this is testing. May be the CTAS is missing?

Reply:
Please see my previous comment.